plugins/keycloak-backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-keycloak-backend",
-  "version": "1.13.3",
+  "version": "1.13.4",
```

Comment:
From my understanding, we don't need this change. Changesets should make this change for us after we merge this pr